#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_ubuntu_paths.dpatch by  <dennis@blackbird.home.kaarsemaker.net>
## Fixed by Per Hansen <spamhans@yahoo.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: New patch generated from freenx 0.6.0-0~seveas1 diff.gz

@DPATCH@
Index: freenx-server-0.7.3.git110520.3884279/node.conf.sample
===================================================================
--- freenx-server-0.7.3.git110520.3884279.orig/node.conf.sample	2012-03-21 11:24:03.030244190 -0300
+++ freenx-server-0.7.3.git110520.3884279/node.conf.sample	2012-03-21 11:27:55.262244605 -0300
@@ -485,7 +485,7 @@
 # Add the nx libraries to LD_LIBRARY_PATH before starting nx agents.
 # WARNING: This will NOT (and should not) affect applications. ONLY Disable
 # this if the nx libraries are in a standard system path (such as /usr/lib)!
-#SET_LD_LIBRARY_PATH="1"
+#SET_LD_LIBRARY_PATH="0"
 
 
 # The command binary for the default window manager. If set it is run when a
@@ -513,15 +513,18 @@
 
 # The key that contains the name of the script that starts a KDE session.
 # It's run when a 'unix-kde' session is requested by the client.
-#COMMAND_START_KDE="/usr/bin/ssh-agent /usr/bin/dbus-launch --exit-with-session /usr/bin/startkde"
+#COMMAND_START_KDE='startkde'
 
 # The key that contains the name of the script that starts a gnome session.
 # It's run when a 'unix-gnome' session is requested by the client.
-#COMMAND_START_GNOME="/usr/bin/ssh-agent /usr/bin/dbus-launch --exit-with-session /usr/bin/gnome-session"
+#COMMAND_START_GNOME='gnome-session --session gnome-fallback'
+
+# Other options is to use unity-2d
+#COMMAND_START_GNOME='gnome-session --session ubuntu-2d'
 
 # The key that contains the name of the script that starts a CDE session.
 # It's run when a 'unix-cde' session is requested by the client.
-#COMMAND_START_CDE=cdwm
+#COMMAND_START_CDE='xfce4-session'
 
 # The key that contains the name of the complete path of command name
 # 'xterm'. It is run when a unix "xterm" session is requested by the
@@ -641,7 +644,7 @@
 # set this to "1".
 # 
 # If nxdesktop cannot be found this is set automatically to "1".
-#ENABLE_EXTERNAL_NXDESKTOP="0"
+#ENABLE_EXTERNAL_NXDESKTOP="1"
 
 # This configuration variable determines if 'rdesktop' command should be run with -k keyboard option
 # or if the keyboard should be autodetected.
@@ -652,4 +655,4 @@
 # set this to "1".
 # 
 # If nxviewer cannot be found this is set automatically to "1".
-#ENABLE_EXTERNAL_NXVIEWER="0"
+#ENABLE_EXTERNAL_NXVIEWER="1"
